whitepaper/whitepaper.md

```diff
@@ -518,51 +518,53 @@ gets replaced by a traditional bank account holding USD.
-| Success Criteria           | Assets have been migrated. An identical history for those assets has been reconstructed on the target blockchain.                                                                                                                                                                                                                                                                                                                                                |
-| Failure Criteria           | 1. It was not migrate the assets. <br> It was not possible to reconstruct the asset history on the target blockchain.                                                                                                                                                                                                                                                                                                                                                                                                          |
-| Prerequisites              | 1. All members belonging to `Consortium A` want to migrate the blockchain. <br> 2. The `Consortium A` controls the source blockchain. <br>3. `Consortium A` has write and execute permissions on the target blockchain<br>
-| Comments                   | An asset is defined as data or smart contracts originating from the source blockchain. <br> This use case relates to use cases implying asset portability (e.g., 2.1) <br> This use case provides blockchain portability, thus reducing costs and fostering blockchain adoption.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-
+| Use Case Attribute Name    | Use Case Attribute Value |
+| -------------------------- | -------------------------------------------- |
+| Use Case Title             | Blockchain Migration |
+| Use Case                   | 1. `User A` runs an application/service on a source blockchain that is either permissioned or permissionless.<br>2. `User A` decides to use another blockchain instance (permissioned or permissionless) to run his/her application. <br>3. `User A` migrates the existing assets and data to the target blockchain.<br>4. `User A` runs the application/service on the target blockchain.
```

Comment:
I am afraid I do not see any "User A" in sequence diagram. 
If the "User A" is not mere a end-user, it should be name as "Administrator A" or something else.
I also want to clarify what is the application described here. If application is intermediates the transfer asset across blockchain ledgers, the statement should be "integrated service application can access to both source blockchain and targeted blockchain".

Reply:
Happy to update the sequence diagram.

Depending on the scenario `User A` can be a mere end-user of a blockchain, an administrator, or even the governance body of a blockchain. So 'User A` is more generic. Any suggestion on preferred terminology?

Application is essentially a DApp. It doesn't do anything to support/perform the migration. For e.g., we could consider application using Cactus to initially talk to source blockchain and after migration still using Cactus to talk to target blockchain. We could also envision application using Cactus only for the migration, and directly talking to target blockchain after migration. Should I use DApp or something else?
Thanks for suggestions.

Reply:
I just wanted to be consistent with texts and sequence diagrams. 
You wrote "Application is essentially a DApp. It doesn't do anything to support/perform the migration. ", but it was your assumption. We may implement pair of DApps(smart contracts) which one can delete transferring asset with creating a proof of burn, and another import the asset referring the proof. I thought that is the reasonable assumption for linking the trust over multiple blockchains.

Reply:
Yes, this is certainly needed to ensure proper linking, and proof-of-burn is already being used in asset migration. However, this isn't essential in data migration unless there's is a need to have only one copy of state that data represents. I was trying not to go into such details in the current write up, especially this is the use case than implementation. Such discussion belongs to Sec. 4 and 5. Hope the changes made in response to next comment now cover this comment too.

Reply:
OK. I can accept them with assuming further works will be done in the future.